### DISCUSSION TITLE
Intel x86 Emulator Accelerator (HAXM installer) r7.6.5

### DISCUSSION THREAD

**Discussion by AngelMartinezAlvarez** [dotnet/maui#13565] [General]
Tengo un problema con la instalacion de Intel x86 Emulator Accelerator (HAXM installer) r7.6.5, a la hora de instalar el archivo y entrar a la ventana de instalación me sale un error el cual dice que mi computadora "no cumple los requisitos del sistema"

**Comment by drasticactions** [comment]
This isn't an issue in MAUI, so I converted it to a discussion item.

Assuming you're trying to set up the android emulator, you don't need HAXM. Read the docs for how to set it up https://learn.microsoft.com/en-us/dotnet/maui/android/emulator/hardware-acceleration?view=net-maui-7.0.
